lib/uniris/transaction_chain/transaction/validation_stamp.ex

```diff
@@ -11,6 +11,7 @@ defmodule Uniris.TransactionChain.Transaction.ValidationStamp do
     :signature,
     :proof_of_work,
     :proof_of_integrity,
+    :proof_of_existence,
```

Comment:
Regarding this name, I have some concerns
- I don't think we understand clearly what it is 
- For other type of transaction this data will be unrelevant. 
- Proof of XXX => Usually are cryptograhic proofs , and here it's just a boolean

I was thinking to adapt this part for a more generic way, as some transaction may be distributed error detection: smart contract creation, smart contract calls, oracles, ...

Of cource we could use a boolean for `oracle_validation` as `contract_validation` but again it may be irrelevant for other transaction validation

So we could imagine a field in which we will find some custom errors, related to the pending transaction itself but aggregated and validated across the miners. It may be more scalable for further implementations.